index.html

```diff
@@ -77,6 +77,16 @@ <h2>Error: Browser not supported</h2>
         </div>
     </div>
 
+    <div class="compat textdecoder" style="display: none;">
+        <img class="logo-compat" src="img/logo.svg?v=[[VERSION]]"/>
+        <div>
+            <h2>Error: Browser not supported</h2>
```

Comment:
Super-nit: If I have a browser that supports neither, I'll get this heading twice now.

Reply:
Yep, but I don't care all that much :wink: